runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -252,6 +252,249 @@ doubleMaxMinHelper(TR::Node *node, TR::CodeGenerator *cg, bool isMaxOp)
+   */
+TR::Register * caseConversionHelper(TR::Node* node, TR::CodeGenerator* cg, bool isToUpper, bool isCompressedString)
+   {
+   TR::Register* sourceRegister = cg->evaluate(node->getChild(0));
+   TR::Register* destRegister = cg->evaluate(node->getChild(1));
+   TR::Register* lengthRegister = cg->gprClobberEvaluate(node->getChild(2));
+
+   TR::Register* addressOffset = cg->allocateRegister();
+   TR::Register* loadLength = cg->allocateRegister();
+   // Loopcounter register for number of 16 byte conversions, when it is used, the length is not needed anymore
+   TR::Register* loopCounter = lengthRegister;
+
+   TR::Register* charBufferVector = cg->allocateRegister(TR_VRF);
+   TR::Register* selectionVector = cg->allocateRegister(TR_VRF);
+   TR::Register* modifiedCaseVector = cg->allocateRegister(TR_VRF);
+   TR::Register* charOffsetVector = cg->allocateRegister(TR_VRF);
+   TR::Register* alphaRangeVector = cg->allocateRegister(TR_VRF);
+   TR::Register* alphaCondVector = cg->allocateRegister(TR_VRF);
+   TR::Register* invalidRangeVector = cg->allocateRegister(TR_VRF);
+   TR::Register* invalidCondVector = cg->allocateRegister(TR_VRF);
+
+   TR::LabelSymbol* fullVectorConversion = TR::LabelSymbol::create(cg->trHeapMemory(), cg);
+   TR::LabelSymbol* doneLabel = TR::LabelSymbol::create(cg->trHeapMemory(), cg);
+   TR::LabelSymbol* handleInvalidChars = TR::LabelSymbol::create(cg->trHeapMemory(), cg);
+   TR::LabelSymbol* loop = TR::LabelSymbol::create(cg->trHeapMemory(), cg);
+
+   TR::Instruction* instruction;
+
+   const int elementSizeMask = (isCompressedString) ? 0x0 : 0x1;    // byte or halfword mask
+   const int sizeOfVector = 16;
+   const bool is64 = TR::Compiler->target.is64Bit();
+   uintptrj_t headerSize = TR::Compiler->om.contiguousArrayHeaderSizeInBytes();
+
+   TR::RegisterDependencyConditions * regDeps = new (cg->trHeapMemory()) TR::RegisterDependencyConditions(0, 14, cg);
+   regDeps->addPostCondition(sourceRegister, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(destRegister, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(lengthRegister, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(addressOffset, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(loadLength, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(charBufferVector, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(selectionVector, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(modifiedCaseVector, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(charOffsetVector, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(alphaRangeVector, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(alphaCondVector, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(invalidRangeVector, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(invalidCondVector, TR::RealRegister::AssignAny);
+
+   if (is64){
```

Comment:
Formatting of braces should be consistent with the rest of the codebase.

Reply:
56171e1